kde-misc/kio-admin/kio-admin-9999.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+ECM_HANDBOOK="forceoptional"
+ECM_TEST="false"
+KDE_ORG_CATEGORY="system"
+KFMIN=5.98.0
+QTMIN=5.15.0
+inherit ecm gear.kde.org
+
+DESCRIPTION="Manage files as administrator using the admin:// KIO protocol."
+HOMEPAGE="https://invent.kde.org/system/kio-admin"
+
+if [[ ${KDE_BUILD_TYPE} = release ]]; then
+	KEYWORDS="~amd64"
+fi
+
+LICENSE="BSD CC0-1.0 FSFAP GPL-2 GPL-3"
+SLOT="6"
+
+DEPEND="
+	>=dev-qt/qtbase-${QTMIN}:6
```

Comment:
no need for any USEdeps here?

Reply:
https://invent.kde.org/search?search=find_package(Qt&nav_source=navbar&project_id=11819&group_id=1561&search_code=true&repository_ref=master

Only Core searched for by find_package. I dont see it linking explicitly to any Qt library.

Reply:
I see linking to Qt6DBus as well as Qt6Gui.